core/web/resolver/ocr.go

```diff
@@ -1,30 +1,29 @@
 package resolver
 
 import (
+	"github.com/graph-gophers/graphql-go"
+
 	"github.com/smartcontractkit/chainlink/core/services/keystore/keys/ocrkey"
 )
 
 type OCRKeyBundle struct {
```

Comment:
Let's this to stay consistent

```suggestion
type OCRKeyBundleResolver struct {
```

We generally make a constructor as well so it's easier to create a new Resolver.

```
NewOCRKeyBundle() *OCRKeyBundleResolver 
```

Reply:
Fixed!